### PR TITLE
Update fallback WNP [fix #14501]

### LINKED ISSUE #14501: Experiment: promote Firefox mobile on fallback WNP
## Description

Currently our evergreen "fallback" what's new page promotes Mozilla Accounts to users who are not signed into an account already, and promotes Firefox mobile to signed-in users. Furthermore, the fallback mobile page is pretty bare-bones and hasn't been updated in a few years.

To try to grow more mobile users we'll do some experimentation on that fallback page, promoting mobile to a percentage of signed-out users (instead of accounts), as well as adding some flare to the mobile page.

Experiment audience parameters and design assets are forthcoming.

Brief doc: https://docs.google.com/document/d/1jbOBVkKQnd1b8N4hr0URzIE-RaQiAnPcayI87YSnKbg/

### PR DESCRIPTION
**Pull request by craigcook**
## One-line summary
Removes the old `index-account.html` file and updates `index.html` to remove the send-to-device widget and show a QR code to all locales, as well as a design update and adding the app store badges. This is the default WNP shown whenever there is no custom template specified; old versions, other locales, etc.

This update began life as a proposed experiment but ultimately we decided to forego the experiment and just simplify the whole thing and ship a redesigned fallback page. Hooray!

## Issue / Bugzilla link
#14501 

## Testing
http://localhost:8000/firefox/99.0/whatsnew/ (old version, all locales)
http://localhost:8000/ja/firefox/126.0/whatsnew/ (recent version but a fallback locale)

### CODE REVIEW

File: bedrock/firefox/views.py [hunk 1]
```diff
@@ -579,14 +579,11 @@ def get_template_names(self):
             else:
                 template = "firefox/whatsnew/index.html"
         else:
-            if show_default_account_whatsnew(version) and ftl_file_is_active("firefox/whatsnew/whatsnew-account"):
```
Comment (alexgibson) [inline bedrock/firefox/views.py:582]: **Suggestion:** if we not longer need it, let's remove the `show_default_account_whatsnew()` function?

File: bedrock/firefox/views.py [hunk 2]
```diff
@@ -403,6 +402,7 @@ class WhatsnewView(L10nTemplateView):
         "firefox/whatsnew/whatsnew-fx126beta-fr.html": ["firefox/whatsnew/whatsnew"],
         "firefox/whatsnew/whatsnew-fx126-na.html": ["firefox/whatsnew/whatsnew"],
         "firefox/whatsnew/whatsnew-fx126-eu.html": ["firefox/whatsnew/whatsnew"],
+        "firefox/whatsnew/whatsnew-fx126-mobile.html": ["firefox/whatsnew/whatsnew-account", "firefox/whatsnew/whatsnew"],
```
Comment (alexgibson) [inline bedrock/firefox/views.py:405]: **Question:** is this unrelated to this PR? I don't see this template anywhere?

File: bedrock/firefox/views.py [hunk 3]
```diff
@@ -379,7 +379,6 @@ class WhatsnewView(L10nTemplateView):
         "firefox/developer/whatsnew.html": ["firefox/developer"],
         "firefox/developer/whatsnew-mdnplus.html": ["firefox/whatsnew/whatsnew-developer-mdnplus"],
         "firefox/nightly/whatsnew.html": ["firefox/nightly/whatsnew", "firefox/whatsnew/whatsnew"],
-        "firefox/whatsnew/index-account.html": ["firefox/whatsnew/whatsnew-account", "firefox/whatsnew/whatsnew"],
```
Comment (alexgibson) [inline bedrock/firefox/views.py:382]: **Question:** Can we remove `firefox/whatsnew/whatsnew-account.ftl`? And also from [pontoon.toml](https://github.com/mozilla/bedrock/blob/main/l10n/configs/pontoon.toml#L194-L196)?

File: bedrock/firefox/templates/firefox/whatsnew/index.html [hunk 4]
```diff
@@ -20,41 +23,28 @@
...
-        <div id="qr-wrapper" class="primary-cta">
-          <img src="{{ static('img/firefox/whatsnew/qrcode-firefox-whatsnew.png') }}" width="250" height="250" alt="">
-        </div>
-      {% endif %}
+<section class="wnp-content-main">
+  <div class="mzp-l-content mzp-t-content-md">
+    <h1 class="wnp-main-title">{{ ftl('whatsnew-s2d-download-firefox-for-your') }}</h1>
+
+    <div class="qr-wrapper">
+      <img class="qr-code" src="{{ static('img/firefox/whatsnew/qrcode-firefox-whatsnew.png') }}" width="180" height="180" alt="{{ ftl('whatsnew-s2d-qr-code-alt') }}">
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/whatsnew/index.html:35]: **Issue:** This `alt` text Fluent string doesn't seem to exist?

File: bedrock/firefox/templates/firefox/whatsnew/index-account.html [hunk 5]
```diff
@@ -1,80 +0,0 @@
...
-          </div>
-        {% endif %}
-        </div>
-      </div>
-  </section>
-{% endblock %}
-
-{% block js %}
-  {{ js_bundle('fxa_product_button') }}
-  {{ js_bundle('firefox_whatsnew_account') }}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/whatsnew/index-account.html:79]: **Suggestion:** remove this JS bundle if it's no longer needed?

File: bedrock/firefox/templates/firefox/whatsnew/index.html [hunk 6]
```diff
@@ -20,41 +23,28 @@
...
+      </li>
+      <li class="ios">
+        {{ apple_app_store_button(href=ios_url, id='app-store-link') }}
+      </li>
+    </ul>
+  </div>
+</section>
 {% endblock %}
 
 {% block js %}
```
Comment (alexgibson) [inline bedrock/firefox/templates/firefox/whatsnew/index.html:50]: **Suggestion:** remove `"js/base/send-to-device.es6.js"` from the `firefox_whatsnew` JS bundle?

File: media/img/firefox/whatsnew/whatsnew_background.svg [hunk 7]
```diff
@@ -1 +0,0 @@
...30.38-926c-150.27-104.6-183.77-300-74.84-436.52s319.06-162.35 469.33-57.75L810-285.73z" fill="none"/></clipPath><linearGradient id="e" x1="762.09" y1="-59.33" x2="617.44" y2="695.2" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#20123a"/><stop offset="1" stop-color="#4b2292"/></linearGradient><clipPath id="f"><path d="M966.42-1281.06c71.87-159.16 269.11-236.08 440.54-171.82s252.14 245.39 180.26 404.54L950.89 360.74C879 519.89 681.78 596.82 510.35 532.55L-1007.46-36.4c-171.43-64.26-252.14-245.38-180.27-404.54S-918.62-677-747.19-612.76l1207.41 452.6z" fill="none"/></clipPath><linearGradient id="g" x1="441.26" y1="112.54" x2="363.07" y2="607.74" gradientUnits="userSpaceOnUse"><stop offset="0" stop-color="#20123a"/><stop offset="1" stop-color="#332869"/></linearGradient></defs><g clip-path="url(#a)"><g clip-path="url(#b)"><path fill="url(#c)" d="M1463.6 148l-630.87 960.61L-938.08-212.73l630.86-960.61L1463.6 148z"/></g><g clip-path="url(#d)"><path fill="url(#e)" d="M680.24-2015L2388 617.22 881.76 1699.07-826.01-933.15 680.24-2015z"/></g><g clip-path="url(#f)"><path fill="url(#g)" d="M-92.43-1803.87l2304.7 2129.18-1192.57 1420.2L-1285-383.67l1192.57-1420.2z"/></g></g></svg>
```
Comment (alexgibson) [inline media/img/firefox/whatsnew/whatsnew_background.svg:1]: **Issue:** this image appears to still be used in the Nightly WNP? http://localhost:8000/en-US/firefox/128.0a1/whatsnew/

**Review by alexgibson — CHANGES_REQUESTED** [review]
A few small issues and nits, but otherwise this is good clean up and it's great to see this very old page get some attention.

Last thing: do we also still need to update/remove functional tests both for this page and the old accounts page?

- https://github.com/mozilla/bedrock/blob/main/tests/functional/firefox/whatsnew/test_whatsnew.py
- https://github.com/mozilla/bedrock/blob/main/tests/functional/firefox/whatsnew/test_whatsnew_60.py

File: bedrock/firefox/views.py (hunk 2 above)
Comment (craigcook) [reply bedrock/firefox/views.py]: This began life as a proposed experiment for 126, but we decided not to do that and just make this the sole fallback so I kind of hastily moved things around and clearly missed a few spots. I should have been more careful.

**Review by alexgibson — CHANGES_REQUESTED** [review]
Updates all look good! Looks like we still need to update / delete the old functional tests though.

**Comment by craigcook** [comment]
Rebased and updated. I think we probably don't need functional tests for this page since it doesn't actually do anything but display, so seems like the basic template view tests should be enough?

File: tests/functional/firefox/whatsnew/test_whatsnew.py [hunk 8]
```diff
@@ -1,28 +0,0 @@
-# This Source Code Form is subject to the terms of the Mozilla Public
-# License, v. 2.0. If a copy of the MPL was not distributed with this
-# file, You can obtain one at https://mozilla.org/MPL/2.0/.
-
-import pytest
-
-from pages.firefox.whatsnew.whatsnew import FirefoxWhatsNewPage
```
Comment (alexgibson) [inline tests/functional/firefox/whatsnew/test_whatsnew.py:7]: Let's please delete the associated page objects that these test files use.

**Review by alexgibson — APPROVED** [review]
r+wc

File: tests/functional/firefox/whatsnew/test_whatsnew.py (hunk 8 above)
Comment (craigcook) [reply tests/functional/firefox/whatsnew/test_whatsnew.py:7]: doh I thought I did... 

`/tests/pages/firefox/whatsnew/whatsnew.py`
`/tests/pages/firefox/whatsnew/whatsnew_60.py`

Are there other references I missed?

Comment (alexgibson) [reply tests/functional/firefox/whatsnew/test_whatsnew.py:7]: Oh sorry, I must have blanked on those! Yes this looks good, merging!

**Pull request merged**